pkg/manager/manager_glob.go

```
@@ -0,0 +1,358 @@
+	// Check if we have a username
+	if username != "" {
+		// Try to find matching rules in username/cursor-rules-collection
+		return handleUsernameGlobPattern(ctx, cursorDir, username, pattern, g)
+	} else {
+		// Try to find matching templates
+		return handleTemplateGlobPattern(cursorDir, pattern, g)
+	}
+}
+
+// handleLocalGlobPattern handles glob patterns for local filesystem.
+func handleLocalGlobPattern(ctx context.Context, cursorDir, pattern string, g glob.Glob) error {
```

golangci-lint (1.64.8)

cognitive complexity 44 of func `handleLocalGlobPattern` is high (> 20)

(gocognit)